salt/states/icinga2.py

```diff
@@ -245,32 +242,37 @@ def request_cert(name, master, ticket, port="5665"):
     cert_request = __salt__["icinga2.request_cert"](name, master, ticket, port)
     if not cert_request["retcode"]:
         ret["comment"] = "Certificate request from icinga2 master executed"
-        ret["changes"]["cert"] = "Executed. Certificate requested: {0}".format(cert)
+        ret["changes"]["cert"] = "Executed. Certificate requested: {}".format(cert)
         return ret
 
-    ret["comment"] = "FAILED. Certificate requested failed with output: {0}".format(
+    ret["comment"] = "FAILED. Certificate requested failed with output: {}".format(
         cert_request["stdout"]
     )
     ret["result"] = False
     return ret
 
 
-def node_setup(name, master, ticket):
+def node_setup(name, master, ticket, master_host=None):
```

Comment:
A unit test needs to be added for this new master_host functionality.  @waynew can often help with writing simple unittests during his Test Clinic on Tuesdays and Thursdays

Reply:
I added unit tests for the new feature.

Reply:
Awesome thank you!